skylarktest/bazeltest.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2018 The Bazel Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Ugly override hack, which redefines DataFile if we assume to run inside
```

Comment:
The solution is no uglier than the problem, so strike the first three words.

Reply:
> The solution is no uglier than the problem, so strike the first three words.

I do disagree since mechanics that work by convention creep me out. Then again, I can live without my three words of cheekiness :)